plugins/quay/src/components/QuayTagDetails/component.tsx

```diff
@@ -0,0 +1,115 @@
+import { Table, TableColumn } from '@backstage/core-components';
+import React from 'react';
+import {
+  Layer,
+  Vulnerability,
+  VulnerabilityListItem,
+  VulnerabilitySeverity,
+} from '../../types';
+import Link from '@material-ui/icons/Link';
+import Warning from '@material-ui/icons/Warning';
+import { SEVERITY_COLORS } from '../../lib/utils';
+
+type QuayTagDetailsProps = {
+  layer: Layer;
+};
+
+// from: https://github.com/quay/quay/blob/f1d85588157eababc3cbf789002c5db521dbd616/web/src/routes/TagDetails/SecurityReport/SecurityReportTable.tsx#L43
+const getVulnerabilityLink = (link: string) => link.split(' ')[0] as string;
+
+const columns: TableColumn[] = [
+  {
+    title: 'Advisory',
+    field: 'name',
+    render: (rowData: any): React.ReactNode => {
+      const row = rowData as Vulnerability;
+      // eslint-disable-next-line no-console
+      console.log('row', row);
+      return (
+        <>
+          {row.Name}
+          {row.Link.trim().length > 0 ? (
+            <a
+              href={getVulnerabilityLink(row.Link)}
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              <Link style={{ marginLeft: '5px' }} />
+            </a>
```

Comment:
Use `Link` from `@backstage/core-components` instead of `a` 

https://github.com/backstage/backstage/blob/master/packages/core-components/src/components/Link/Link.tsx

https://backstage.io/storybook/?path=/story/navigation-link--pass-props